apps/root/src/pages/strategy-guardian-detail/strategy-management/index.tsx

```diff
@@ -35,25 +37,30 @@ const StrategyManagement = ({ chainId, strategyGuardianId }: StrategyManagementP
     if (strategy?.asset) dispatch(setAsset(strategy.asset));
   }, [strategy?.asset]);
 
+  const hasAssetDelayedWithdrawal = strategy?.asset.withdrawTypes.includes(WithdrawType.DELAYED);
+
   return (
     <StyledBackgroundPaper sx={{ height: height }}>
       <Typography variant="h4Bold">{strategy?.farm.name || <Skeleton width="6ch" variant="text" />}</Typography>
-      <UnderlinedTabs value={tab} onChange={(_, val: number) => setTab(val)}>
-        <Tab
-          label={
-            <Typography variant="bodyRegular" color="inherit">
-              {<FormattedMessage description="earn.strategy-management.tabs.deposit" defaultMessage="Deposit" />}
-            </Typography>
-          }
-        />
-        <Tab
-          label={
-            <Typography variant="bodyRegular" color="inherit">
-              <FormattedMessage description="earn.strategy-management.tabs.withdraw" defaultMessage="Withdraw" />
-            </Typography>
-          }
-        />
-      </UnderlinedTabs>
+      <ContainerBox>
```

Comment:
what about gaps and alignment?

Reply:
I think no gap or alignment is needed here. Gap = 0 ensures the underlined from tabs continue with the borderBottom of this warning, and alignItems = stretch ensures that the height of both siblings is the same (and gets this line continuity)